build/images/test/start_ovs_netdev

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+# Copyright 2022 Antrea Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -euo pipefail
+
+/usr/share/openvswitch/scripts/ovs-ctl --system-id=random start --db-file=/var/run/openvswitch/conf.db
```

Comment:
1. The script doesn't do anything related to netdev, is it still proper to use this name?
2. test-integration still passes an argument which doesn't exist: "start_ovs_netdev --start-ovs-only"

I think there is no point to keep this script, test-integration could call "/usr/share/openvswitch/scripts/ovs-ctl --system-id=random start --db-file=/var/run/openvswitch/conf.db" directly.